production/hoprd-operator/application.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: argoproj.io/v1alpha1
 kind: Application
 metadata:
   name: hoprd-operator
-  namespace: staging-operator
+  namespace: production-operator
```

Comment:
Please notice that at this moment, only one instance of the operator can be running. I would suggest not to name it's namespace as staging or production, as it does not matter in that sense

Reply:
I would say that the problem is that you are using the same cluster for different needs (Staging and production). This Is not a good practice at all.

Reply:
it's not ideal to use the same key always, right now not a big problem but IMO we should set it up differently so we can do that